resources/function_help/json/map_prefix_keys

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "map_prefix_keys",
+  "type": "function",
+  "groups": ["Maps"],
+  "description": "Returns a map with all keys prefixed by prefix.",
```

Comment:
... prefixed by a given string might be a better function description.

Same for parameter description.